tough-kms/src/lib.rs

```diff
@@ -71,7 +71,7 @@ impl fmt::Debug for KmsKeySource {
         f.debug_struct("KmsKeySource")
             .field("key_id", &self.key_id)
             .field("profile", &self.profile)
-            .finish()
+            .finish_non_exhaustive()
```

Comment:
The warning for these is:

```
warning: manual `Debug` impl does not include all fields
```

Using `finish_non_exhaustive()` indicates the entire struct is not intentionally used for the debug representation.